src/test/validatorTest.ts

```diff
@@ -1910,5 +1910,19 @@ describe('validator', () => {
             expect(result).to.have.deep.property('templateValid', true);
             expect(result['errors']['crit']).to.have.lengthOf(0);
         });
+
```

Comment:
Lot of change in this file due to whitespace at the end of line automatically removed by my IDE. Sorry about that.